tests/integration/synapseclient/integration_test_Entity.py

```diff
@@ -499,21 +499,22 @@ def test_create_or_update_project(syn, project, schedule_for_cleanup):
 
 
 def test_download_file_false(syn, project, schedule_for_cleanup):
-    RENAME_SUFFIX = "blah"
+    RENAME_SUFFIX = "blah" + str(uuid.uuid4())
 
     # Upload a file
     filepath = utils.make_bogus_binary_file()
     schedule_for_cleanup(filepath)
     schedule_for_cleanup(filepath + RENAME_SUFFIX)
-    file = File(filepath, name="SYNR 619", parent=project)
+    file_name = "SYNR-619-" + str(uuid.uuid4())
+    file = File(filepath, name=file_name, parent=project)
     file = syn.store(file)
 
     # Now hide the file from the cache and download with downloadFile=False
     os.rename(filepath, filepath + RENAME_SUFFIX)
     file = syn.get(file.id, downloadFile=False)
 
     # Change something and reupload the file's metadata
-    file.name = "Only change the name, not the file"
+    file.name = "Only change the name, not the file" + str(uuid.uuid4())
```

Comment:
Does this creates a new uuid4 string?

Reply:
Yes:
```
def uuid4():
    """Generate a random UUID."""
    return UUID(bytes=os.urandom(16), version=4)
```

Reply:
@BryanFauble Right, were we supposed to be printing out `filepath + RENAME_SUFFIX`

Reply:
In this test it's doing:
```
    # Change something and reupload the file's metadata
    file.name = "Only change the name, not the file" + str(uuid.uuid4())
    reupload = syn.store(file)
    assert reupload.path is None, "Path field should be null: %s" % reupload.path

    # This should still get the correct file
    reupload = syn.get(reupload.id)
    assert filecmp.cmp(filepath + RENAME_SUFFIX, reupload.path)
    assert reupload.name == file.name
```
My thought is that I want to avoid statically referencing anything - hence the addition of the UUID.

I'm not sure what you mean about `print out a random string`

Reply:
Ohhhh i understand now. Sorry about that - makes sense!